### ISSUE TITLE
Send Audio

### ISSUE THREAD

**Issue by thiagoamarante** [danielcardeenas/sulla#293]
guys, how do you send an audio?

**Comment by robertsLando** [comment]
Did you tried sendFile api?

---

Daniel - Software Engineer

Support me at:
    Github sponsors

**Comment by thiagoamarante** [comment]
yes, and it works. but it appears for you to download. Unlike an audio that you send at the same time.

**Comment by danielcardeenas** [comment]
Sending audio clip as a recording is not supported. Right now you can only send audio files

**Comment by thiagoamarante** [comment]
@danielcardeenas I'm checking how it handles audio in the store.
__x_type: "ptt"
__x_isMedia: false
